src/components/VideoChatButtonAndMenu.js

```diff
@@ -0,0 +1,115 @@
+
+class VideoChatButtonAndMenu extends Component {
+    menuItemData = [
+        {
+            icon: ZoomIcon,
+            text: 'Zoom',
+            onPress: () => openURLInNewTab(CONST.NEW_ZOOM_MEETING_URL),
+        },
+        {
+            icon: GoogleMeetIcon,
+            text: 'Google Meet',
+            onPress: () => openURLInNewTab(CONST.NEW_GOOGLE_MEET_MEETING_URL),
+        },
+    ].map(item => ({
+        ...item,
+        onPress: () => {
+            item.onPress();
+            this.toggleVideoChatMenu();
+        },
+    }));
+
+    constructor(props) {
+        super(props);
+
+        this.toggleVideoChatMenu = this.toggleVideoChatMenu.bind(this);
+        this.measureVideoChatIconPosition = this.measureVideoChatIconPosition.bind(this);
+        this.videoChatIconWrapper = null;
+
+        this.state = {
+            isVideoChatMenuActive: false,
+            videoChatIconPosition: {x: 0, y: 0},
+        };
+    }
+
+    /**
+     * Toggles the state variable isVideoChatMenuActive
+     */
+    toggleVideoChatMenu() {
+        this.setState(prevState => ({
+            isVideoChatMenuActive: !prevState.isVideoChatMenuActive,
+        }));
+    }
+
+    /**
+     * This gets called onLayout to find the cooridnates of the wrapper for the video chat button.
+     */
+    measureVideoChatIconPosition() {
+        if (this.videoChatIconWrapper) {
+            this.videoChatIconWrapper.measureInWindow((x, y) => this.setState({
+                videoChatIconPosition: {x, y},
+            }));
+        }
+    }
+
+    render() {
+        return (
+            <>
+                <View
+                    ref={el => this.videoChatIconWrapper = el}
+                    onLayout={this.measureVideoChatIconPosition}
+                >
+                    <Pressable
+                        onPress={() => {
+                            this.toggleVideoChatMenu();
+                        }}
+                        style={[styles.touchableButtonImage, styles.mr0]}
+                    >
+                        <Icon
+                            src={Phone}
+                            fill={this.state.isVideoChatMenuActive
+                                ? themeColors.heading
+                                : themeColors.icon}
+                        />
+                    </Pressable>
+                </View>
+                <Popover
+                    onClose={this.toggleVideoChatMenu}
+                    isVisible={this.state.isVideoChatMenuActive}
+                    anchorPosition={{
+                        left: this.state.videoChatIconPosition.x - 150,
+                        top: this.state.videoChatIconPosition.y + 40,
+                    }}
+                    animationIn="fadeInDown"
+                    animationOut="fadeOutDown"
+                >
+                    {this.menuItemData.map(({icon, text, onPress}) => (
+                        <MenuItem
+                            style={[styles.mr3]}
```

Comment:
NAB no need for square brackets when it's just one style
```suggestion
                            style={styles.mr3}
```